contracts/ERC20SimpleSwap.sol

```diff
@@ -114,10 +119,10 @@ contract SimpleSwap {
     paidOut[beneficiary] = paidOut[beneficiary].add(totalPayout);
     /* do the actual payments */
 
-    recipient.transfer(totalPayout.sub(callerPayout));
+    token.transfer(recipient, totalPayout.sub(callerPayout));
```

Comment:
Maybe we should document somewhere that this contract expects the token to throw on error.

Reply:
I wrapped all `token.transfer`s in in `require` statements.